### DISCUSSION TITLE
Did Tuya remove the free Basic edition plan?

### DISCUSSION THREAD

**Discussion by Panoramiac** [codetheweb/tuyapi#624] [Q&A]
I currently try to setup local Tuya and fail to get the device IDs as my IoT trial plan expired. I can not find the free Basix plan to book on the purchase page:
![image](https://user-images.githubusercontent.com/48438802/224483749-882bb603-615e-4af4-a348-9b52ecc3a1b9.png)

So my quesion is, if this is no longer available or is the problem on my side as I might do something wrong?

Thanks,
Tobias

**Comment by Apollon77** [comment]
It shows price 0,00 this is still free, or?

**Reply by Panoramiac** [reply]
This is the trial for a month, so it is limited and for me currently expired. I am waiting for an approval to extend it. So currently the interface can not be used any more.

**Reply by Apollon77** [reply]
Yes, and in fact you need to prolong this once every month ... mabe you can extend 3 months too or such for free ... It is as it is
